src/Unused/Parser.hs

```diff
@@ -39,7 +37,7 @@ parseTermMatch = do
     colonSep = do { void $ try $ char ':' }
 
 termChars :: Parser Char
-termChars = alphaNum <|> char '_' <|> char '!' <|> char '?' <|> char '='
+termChars = alphaNum <|> char '_' <|> char '!' <|> char '?' <|> char '=' <|> char '>' <|> char '<'
```

Comment:
I think `choice [a, b, c, d]` is equivalent to (and IMO better than) `a <|> b <|> c <|> d`